### PR TITLE
feat: 追蹤看更多 experience 的數據

### PR DESCRIPTION
**Pull request by barry800414**
## 這個 PR 是？ 

- 追蹤單篇文章頁面，點擊看更多 experience 區塊的數據

## 我應該如何手動測試？ 

- [ ] 單篇 experience 下方點擊面試或評價區塊，有送 google analytics 事件
Screenshot 2024-10-26 at 4 02 21 PM
Screenshot 2024-10-26 at 4 02 28 PM

- [ ] 公司或職稱頁面，下方點擊面試或評價區塊，不會送 `click_read_more_experience` 事件。應該只會有 page view
Screenshot 2024-10-26 at 4 03 01 PM

### CODE REVIEW

File: src/components/CompanyAndJobTitle/InterviewExperiences/ExperienceEntry.js [hunk 1]
```diff
@@ -31,8 +31,9 @@ const ExperienceEntry = ({
   },
   size,
   canView,
+  onClick = () => {},
```
Comment (mark86092) [inline src/components/CompanyAndJobTitle/InterviewExperiences/ExperienceEntry.js:34]: Add the default value to defaultProps

Although defaultProps is considered duplicated with default argument and is not encouraged, we are still use the defaultProps.

I think we could one day upgrade the react dependency and revamp the whole repo to use the default argument

Comment (peteranny) [reply src/components/CompanyAndJobTitle/InterviewExperiences/ExperienceEntry.js:34]: 建議 default to null 甚至 undefined 就好了，因為 non-required

Comment (barry800414) [reply src/components/CompanyAndJobTitle/InterviewExperiences/ExperienceEntry.js:34]: Thanks for suggestion, updated it !

Comment (mark86092) [reply src/components/CompanyAndJobTitle/InterviewExperiences/ExperienceEntry.js:34]: 咦，這個還是沒看到 default to null 耶

Comment (barry800414) [reply src/components/CompanyAndJobTitle/InterviewExperiences/ExperienceEntry.js]: 只改到一邊，兩邊都改了！

**Review by mark86092 — APPROVED** [review]

**Review by peteranny — APPROVED** [review]

**Pull request merged**